Frontend/linkedout/src/App.js

```diff
@@ -18,8 +18,9 @@ const router = createBrowserRouter([
     errorElement: <NotFoundPage />
   },
   {
-    path:"/RegistrationPage",
-    element: <RegistrationPage />
+    path: '/register',
```

Comment:
make sure to adjust in the HomePage.tsx file to have the register button navigate to the appropriate path

Reply:
I think its correct?